source/faq.txt

```diff
@@ -30,7 +30,7 @@ destination cluster as long as the source cluster doesn’t include a namespace
 with the same name. If you write to a synced namespace before issuing a 
```

Comment:
Above where it says:

> You can perform reads during synchronization if ``canCommit`` is ``true``. 
However, the data that you read is 
:term:`eventually consistent <eventual consistency>`, meaning that you might not 
always read the latest writes.

Some clarifications: you can read at any time, but you may not get the latest writes due to eventual consistency. If you wait until committed, you will get the up to date writes.

Reply:
updated